sdks/android/Mentat/gradle.properties

```diff
@@ -10,6 +10,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 org.gradle.jvmargs=-Xmx1536m
+org.gradle.configureondemand=false
```

Comment:
Is this really necessary?  This sucks, because always configuring all projects is very expensive as sub-projects multiply.